main/admin/course_update_import.php

```diff
@@ -102,17 +102,39 @@ function validateCourseData(array $courses): array
 function updateCourse(array $courseData, int $courseId): void
 {
     $courseTable = Database::get_main_table(TABLE_MAIN_COURSE);
-    $params = [
-        'title' => $courseData['Title'],
-        'course_language' => $courseData['Language'],
-        'category_code' => $courseData['CourseCategory'],
-        'visual_code' => $courseData['Code'],
+    $params = ['visual_code' => $courseData['Code']];
```

Comment:
The visual_code should not be changed through this CSV import, you can just remove that line.

Reply:
ok, it is removed that line now.